Sorting/CountSort.php

```diff
@@ -1,30 +1,40 @@
 <?php
 
 /**
- * @param $array
- * @return mixed
+ * Sorts an array of integers using the Counting Sort algorithm.
+ *
+ * @param array $array The input array to be sorted.
+ * @param int $min The minimum value in the input array.
+ * @param int $max The maximum value in the input array.
+ * @return array The sorted array.
  */
-function countSort($array)
+function countSort(array $array, int $min, int $max)
```

Comment:
This was changed in a recently merged PR.  These arguments are no longer needed - the min and max are calculated within this function now.